localstack/services/cloudformation/models/apigateway.py

```diff
@@ -130,9 +130,14 @@ def _create(resource_id, resources, resource_type, func, stack_name):
             resource = resources[resource_id]
             props = resource["Properties"]
 
+            tag_list = props.get("Tags", [])
```

Comment:
small cosmetic nit: I'd inline `tag_list` on the dictionary comprehension and reword `tag_map` to `tags`, something around these lines:

 ```suggestion
       tags = {tag["Key"]: tag["Value"] for tag in props.get("Tags", [])}
```